lib/types.js

```diff
@@ -1626,15 +1626,22 @@ function EnumType(schema, opts) {
   if (!Array.isArray(schema.symbols) || !schema.symbols.length) {
     throw new Error(f('invalid enum symbols: %j', schema.symbols));
   }
+
   this.symbols = Object.freeze(schema.symbols.slice());
   this._indices = {};
+  this.defaultValue = undefined;
```

Comment:
Can you switch to `defaultSymbol`? It would be slightly more specific and consistent.